src/microsim/psf.py

```diff
@@ -350,12 +352,14 @@ def make_psf(
     space: SpaceProtocol,
     channel: OpticalConfig,
     objective: ObjectiveLens,
+    em_wvl: Quantity | None = None,
```

Comment:
i think you've done the right thing here for this PR... but it makes me want to reconsider the whole signature of taking in an `OpticalConfig` object (later)